jupyter/minimal/ubi8-python-3.8/requirements.in

```diff
@@ -1 +1,60 @@
+# Minimal notebook requirements
 jupyterlab==3.2.4
+jupyter_server==1.4.1
+notebook==6.4.12
+setuptools==65.4.0
+pip==22.2.2
+
+# JupyterLab extensions
+# A JupyterLab extension for version control using git
```

Comment:
We don't have to add a description per extension, it can be useful for newcomers to understand what every extension does but worsens readability on a daily-basis